### PR TITLE
[ao][docs] Clarify operator support for quantization

### PR DESCRIPTION
**Pull request by raghuramank100**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63270
* #63582

Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30658654](https://our.internmc.facebook.com/intern/diff/D30658654)

### CODE REVIEW

File: docs/source/quantization.rst [hunk 1]
```diff
@@ -147,23 +147,54 @@ The following table compares the differences between Eager Mode Quantization and
...
+
++-----------------+--------------------+------------------+
+|                 |Static             |      Dynamic      |
+|                 |Quantization       | Quantization      |
++-----------------+-------------------+-------------------+
+|nn.Linear        | Y                 | Y                 |
+|nn.Conv1d/2d/3d  | Y                 | N                 |
++-----------------+-------------------+-------------------+
+|nn.LSTM          | N                 | Y                 |
+|nn.GRU           |                   |                   |
```
Comment (vkuzo) [inline docs/source/quantization.rst:174]: nit: specify for each line, to stay consistent with L170:L171?

File: docs/source/quantization.rst [hunk 2]
```diff
@@ -147,23 +147,54 @@ The following table compares the differences between Eager Mode Quantization and
...
++-----------------+-------------------+-------------------+
+|nn.RNNCell       | N                 | Y                 |
+|nn.GRUCell       |                   |                   |
+|nn.LSTMCell      |                   |                   |
```
Comment (vkuzo) [inline docs/source/quantization.rst:178]: same

File: docs/source/quantization.rst [hunk 3]
```diff
@@ -147,23 +147,54 @@ The following table compares the differences between Eager Mode Quantization and
...
++-----------------+-------------------+-------------------+
+|nn.EmbeddingBag  | Y* (activations   |                   |
+|                 | are in fp32)      | Y                 |
++-----------------+-------------------+-------------------+
+|nn.Embedding     | Y                 | N                 |
++-----------------+-------------------+-------------------+
+|nn.MHA           |* Supported via    |                   |
```
Comment (vkuzo) [inline docs/source/quantization.rst:185]: nit: can we spell out the name (not everyone will know what MHA means), and should we consider not saying this is supported yet until there is a clear long term strategy and clear documentation?

**Review by vkuzo — APPROVED** [review]

File: docs/source/quantization.rst (hunk 3 above)
Comment (raghuramank100) [reply docs/source/quantization.rst:185]: Agreed, will change that.

**Comment by raghuramank100** [comment]
@raghuramank100 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30658654).

**Pull request closed without merging**